docs/sources/administration/home_dashboard.md

```diff
@@ -0,0 +1,36 @@
++++
+title = "How to replace the default home dashboard"
+description = "How to replace the default home dashboard"
+keywords = ["grafana", "configuration", "documentation", "home"]
+type = "docs"
+[menu.docs]
+name = "Home Dashboard"
+identifier = "home-dashboard"
+parent = "admin"
+weight = 100
++++
+
+# How to replace the default home dashboard
+
+You can easily replace the default homepage of Grafana in two different ways.
+
+## Preferences
+
+You can choose any of your existing Grafana dashboards to replace the default home dashboard in the preferences. First, you need to star the dashboard that you want to use. Then, in `Configuration > Preferences`, you can select one of your starred dashboards in the `Home Dashboard` input. It will set it for all the users in your organization.
```

Comment:
```suggestion
You can choose any of your existing Grafana dashboards to replace the default home dashboard in the preferences. 

1. Star the dashboard that you want to use. 
1. In **Configuration > Preferences**, select one of your starred dashboards in the **Home Dashboard** field. 

Changing the preference changes the home dashboard for all the users in your organization.
```

Reply:
These instructions are incomplete. WHERE do you star the dashboard? How do you do it?

Also, you need to make clear before the instructions who this change will affect and how to affect just you vs. the whole organization.